modules/lfs/http_client.go

```diff
@@ -30,7 +29,7 @@ type HTTPClient struct {
 
 // BatchSize returns the preferred size of batchs to process
 func (c *HTTPClient) BatchSize() int {
-	return httpBatchSize
+	return setting.LFSClient.BatchSize
```

Comment:
I realized you were talking about this, thanks that was lost in the config refactor. Put it back.